monitoring/dashboard.py

```diff
@@ -0,0 +1,412 @@
+        expr="\n".join([
+            'sum(rate(http_requests_total{namespace="${namespace}", job="${job}", code=~"2.."}[$__rate_interval])) * 100',
+            "   /",
+            'sum(rate(http_requests_total{namespace="${namespace}", job="${job}"}[$__rate_interval]))',
+        ]),
+        instant=True,
+        legendFormat="Success rate",
+    )],
+    thresholds=[
+        Threshold("red",    0, 0.0),
+        Threshold("orange", 1, 80.0),
+        Threshold("green",  2, 90.0),
+    ],
+)
+
+bucketsCounter = Stat(
+    title="Buckets",
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="background",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(cloud_server_number_of_buckets{namespace="${namespace}",job="${job}"})',
+    )],
+    thresholds=[
+        Threshold("dark-purple", 0, 0.0),
+    ],
+)
+
+objectsCounter = Stat(
+    title="Objects",
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="background",
+    format=UNITS.SHORT,
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(cloud_server_number_of_objects{namespace="${namespace}",job="${job}"})',
+    )],
+    thresholds=[
+        Threshold("dark-purple", 0, 0.0),
+    ],
+)
+
+dataDiskStorage = TimeSeries(
+    title="Data disk storage",
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=30,
+    lineInterpolation="smooth",
+    targets=[
+        Target(
+            expr='sum(delta(cloud_server_data_disk_total{namespace="${namespace}",job="${job}"}[$__rate_interval]))',
+            legendFormat='Total',
+        ),
+        Target(
+            expr='sum(delta(cloud_server_data_disk_free{namespace="${namespace}",job="${job}"}[$__rate_interval]))',
+            legendFormat='Free',
+        ),
+        Target(
+            expr='sum(delta(cloud_server_data_disk_available{namespace="${namespace}",job="${job}"}[$__rate_interval]))',
+            legendFormat='Available',
+        )
+    ],
+)
+
+def http_status_panel(title: str, code: str) -> Stat:
+    return Stat(
+        title=title,
+        dataSource="${DS_PROMETHEUS}",
+        colorMode="background",
+        decimals=0,
+        format=UNITS.SHORT,
+        noValue="0",
+        reduceCalc="lastNotNull",
+        targets=[Target(
+            expr='sum(increase(http_requests_total{namespace="${namespace}",job="${job}",code=' + code + "}[$__rate_interval]))",
+        )],
+        thresholds=[Threshold("semi-dark-blue", 0, 0.)],
+    )
+status200 = http_status_panel(title="Status 200", code='"200"')
+status403 = http_status_panel(title="Status 4xx", code='~"4.."')
+status5xx = http_status_panel(title="Status 5xx", code='~"5.."')
+
+activeRequests = Stat(
+    title="Active requests",
+    dataSource="${DS_PROMETHEUS}",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(http_active_requests{namespace="${namespace}", job="${job}"})',
+    )],
+    thresholds=[
+        Threshold("green", 0, 0.0),
+        Threshold("red",   1, 80.0),
```

Comment:
this is same as Vault, just an arbitrary limit I guess
It should generally stay low since it counts "in flight" requests only; but maybe 80 is normal in a loaded system.
Since it just changes the color of the panel, I guess this should be fine anyway : what do you think?